src/column/reader.rs

```diff
@@ -339,127 +369,170 @@ mod tests {
 
   #[test]
   fn test_read_plain_int32() {
-    let mut tester = ColumnReaderTester::<Int32Type>::new();
     let primitive_type = SchemaType::primitive_type_builder("a", PhysicalType::INT32)
       .with_repetition(Repetition::REQUIRED)
       .with_logical_type(LogicalType::INT_32)
       .with_length(-1)
       .build()
       .expect("new_primitive_type() should be OK");
-    let descr = ColumnDescriptor::new(
-      Rc::new(primitive_type), None, MAX_DEF_LEVEL, MAX_REP_LEVEL, ColumnPath::new(Vec::new()));
-    tester.test_plain(Rc::new(descr), NUM_PAGES, NUM_LEVELS, 16, ::std::i32::MIN, ::std::i32::MAX);
+    let desc = Rc::new(ColumnDescriptor::new(
+      Rc::new(primitive_type), None, MAX_DEF_LEVEL, MAX_REP_LEVEL, ColumnPath::new(Vec::new())));
+
+    let mut tester = ColumnReaderTester::<Int32Type>::new();
+    tester.test_plain_v1(desc.clone(), NUM_PAGES, NUM_LEVELS, 16, ::std::i32::MIN, ::std::i32::MAX);
+    let mut tester = ColumnReaderTester::<Int32Type>::new();
+    tester.test_plain_v2(desc.clone(), NUM_PAGES, NUM_LEVELS, 16, ::std::i32::MIN, ::std::i32::MAX);
   }
 
   #[test]
   fn test_read_plain_int32_uneven() {
-    let mut tester = ColumnReaderTester::<Int32Type>::new();
     let primitive_type = SchemaType::primitive_type_builder("a", PhysicalType::INT32)
```

Comment:
Can we replace this with `get_test_int32_type()` ? I think I missed these in the last commit.